packages/adapters/cache/src/lib/caches/transactions.ts

```diff
@@ -39,7 +39,7 @@ export class TransactionsCache extends Cache {
     const status = this.data.scanStream({
       match: `${txid}`,
     });
-    return new Promise((res, rej) => {
+    return new Promise((res, _) => {
```

Comment:
You don't need to include it if you're not going to use it. That being said if it exits in anyway that is not handled by your `res` then it will just hang

Reply:
oops right